PROTOCOL.md

```diff
@@ -1869,6 +1872,8 @@ The request body should be a JSON string containing the following two optional f
 - **limitHint** (type: Int32, optional): an optional limit number. It’s a hint from the client to tell the server how many rows in the table the client plans to read. The server can use this hint to return only some files by using the file stats. For example, when running `SELECT * FROM table LIMIT 1000`, the client can set `limitHint` to `1000`.
   - Applying `limitHint` is **BEST EFFORT**. The server may return files containing more rows than the client requests.
 
+- **version** (type: Int32, optional): an optional version number. If set, will return files as of the specified version of the table. This is only supported on tables with change data feed (cdf) enabled. 
```

Comment:
I remember in your implementation PR this is a `long` (int64).

Reply:
updated.

`limitHint` is actually also int64 in protocol. I was following it.

But I see that `Query Table Version` is returning table version as long, so int64 makes sense.